persistence/proto/persistence_config.proto

```diff
@@ -9,4 +9,9 @@ message PersistenceConfig {
   string block_store_path = 3;
   string tx_indexer_path = 4;
   string trees_store_dir = 5;
+  int32 max_conns_count = 6;
```

Comment:
Consider adding a comment above the proto saying "See https://pkg.go.dev/time#ParseDuration for expected string time parameters"

Reply:
@Prajjawalk Wdyt?